pytorch_lightning/utilities/cli.py

```diff
@@ -889,7 +889,7 @@ def instantiate_class(args: Union[Any, Tuple[Any, ...]], init: Dict[str, Any]) -
 
 
 def _get_short_description(component: object) -> Optional[str]:
-    parse = import_docstring_parse("LightningCLI(run=True)")
+    parse, _ = import_docstring_parse("LightningCLI(run=True)")
```

Comment:
This was a breaking change. @mauvilsa you might want to have jsonargparse expose the `get_short_description` function